src/demos/calcite-radio-button.html

```diff
@@ -6,6 +6,44 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge" />
     <title>Calcite Radio Button</title>
     <style>
+      .parent-flex,
+      .header {
+        display: flex;
+        padding: 12px 0;
+        width: 70%;
+        color: var(--calcite-ui-text-3);
+        font-family: var(--calcite-sans-family);
+        font-weight: var(--calcite-font-weight-medium);
+
+      }
+
+      .parent-flex {
+        font-size: var(--calcite-font-size-0);
+      }
+
+      .header {
+        font-size: var(--calcite-font-size-4);
```

Comment:
@ManahariDahal when are you planning to start combining repeated classes into src/demos/_assets/demos.css for re-use?

Reply:
Even though the `css` code looks similar, there are few minor differences. Once every demo is deployed, I would work on a single branch combining all the similar classes.

Reply:
Okay cool that works for me